gridstatus/miso.py

```diff
@@ -1323,35 +1395,86 @@ def _get_constraint_header_dates_from_excel(
         )
         return market_date, publish_date
 
+    @support_date_range(frequency="DAY_START")
+    def get_look_ahead_hourly(
+        self,
+        date: str | pd.Timestamp,
+        end: str | pd.Timestamp | None = None,
+        verbose: bool = False,
+    ) -> pd.DataFrame:
+        if date == "latest":
+            return self.get_look_ahead_hourly(date="today", verbose=verbose)
 
-def add_interval_end(df: pd.DataFrame, duration_min: int) -> pd.DataFrame:
-    """Add an interval end column to a dataframe
+        url = f"https://docs.misoenergy.org/marketreports/{date.strftime('%Y%m%d')}_sr_la_rg.csv"
+        logger.info(f"Downloading look-ahead hourly data from {url}")
+        publish_date = pd.read_csv(
```

Comment:
Is the `publish_date` always the date in the file name? In that case, I think we could do `publish_time = date.normalize()`.